src/js/claims-status/components/AppealListItemV2.jsx

```diff
@@ -50,6 +50,7 @@ export default function AppealListItem({ appeal }) {
         <div className={`status-circle ${appeal.attributes.active ? 'open' : 'closed'}`}/>
         <p><strong>Status:</strong> {getStatusContents(status.type, status.details).title}</p>
       </div>
+      {appeal.attributes.description && <span><strong>Issues on appeal:</strong> {appeal.attributes.description}</span>}
```

Comment:
Maybe not strictly necessary, but should this vary based on whether there is one issue or many issues? This could come from `appeal.attributes.issues.length`

Reply:
Vary in what way? My understanding is the caseflow api (is that the one, @cmgiven?) will take care of constructing the description based on some logic, but it'll take into consideration of the number of issues on appeal.

Reply:
I.e., do we want the text to change from 'Issue on appeal' to 'Issues on appeal' based on how many issues there actually are?

Reply:
Ahhhh, clever. Yeah, I think that makes sense. :+1:

Pinging @gnakm for visibility.